ebmlite/encoding.py

```diff
@@ -22,28 +23,28 @@
 # ==============================================================================
 
 # If no length is given, use the platform's size of a float.
-DEFAULT_FLOAT_SIZE = 4 if sys.maxsize <= 2147483647 else 8
-
-LENGTH_PREFIXES = [0,
-                   0x80,
-                   0x4000,
-                   0x200000,
-                   0x10000000,
-                   0x0800000000,
-                   0x040000000000,
-                   0x02000000000000,
-                   0x0100000000000000
-                   ]
+DEFAULT_FLOAT_SIZE: int = 4 if sys.maxsize <= 2147483647 else 8
+
+LENGTH_PREFIXES: List[int] = [0,
```

Comment:
Do internally-used constants need typing?

Reply:
I probably got a little over-zealous when I originally did this, honestly.